### DISCUSSION TITLE
Stats API breakdown endpoint to get realtime and "30m"?

### DISCUSSION THREAD

**Discussion by luciandavidescu** [plausible/analytics#1572] [Anything Else]
First of all, how could I get via the API the realtime "top pages" list that's dispalyed the console? 
I see that 'realtime' comes up in code with other time periods but it's not documented in reference, and when i curl just "(...)?period=realtime(...)" it says "error / invalid period `realtime`, so I suppose some other parameter is also needed.

Second of all, is there a way to also retrieve a breakdown based on the "pageviews (last 30 min)" property that comes up in the summary of every page?
My use case for that is to make a "trending" section, and realtime is a bit too volatile for that, while going back a bit cuts out the noise and adds relevance. (right now I use log parsing for this job and it's one maintenance headache I'd happily leave behind).

**Comment by ukutaht** [comment]
Yeah currently the realtime capabilities are only on the dashboard, not so much in the API.

What would the ideal time range be for you? 30 minutes? 6 hours?

It might make sense to come up with some more flexible ways to define date/time ranges in the API. It can definitely be added but I can't make any promises as to when we have the time to tackle it.

**Reply by luciandavidescu** [reply]
Currently I clear log entries older than an hour, every 10 minutes revolving. But 30 minutes flat would be just as good – I was hoping maybe there was some undocumented parameter somewhere.

**Reply by bactisme** [reply]
I don't think we need a different definition for what is a realtime visitors between the dashboard and the API. The same value (30min), available through the API and the dashboard would be great and sufficient.
